ci/scripts/vale.sh

```diff
@@ -1,4 +1,6 @@
 #!/bin/bash
 
-go get github.com/ValeLint/vale
```

Comment:
We used to use Go to download Vale, but there currently isn't a way to use `go get` and specify a version. Instead, we now download the executable from the releases page per the repo's [README](https://github.com/errata-ai/vale#installation).